models/src/test/scala/coop/rchain/models/blockImplicits.scala

```diff
@@ -35,7 +35,7 @@ object blockImplicits {
   val bondGen: Gen[Bond] = for {
     byteArray <- listOfN(Validator.Length, arbitrary[Byte])
     validator = ByteString.copyFrom(byteArray.toArray)
-    stake     <- arbitrary[Long]
+    stake     <- Gen.chooseNum(1L, 1024L)
```

Comment:
Question: Are you changing this because of negative numbers?

Reply:
Yes, bonds should not be negative. Also if max is `Long.max` - this might cause overflow when summing two stakes, thats why I set `1024L` here